.kokoro/release/stage.sh

```diff
@@ -34,6 +34,8 @@ excludedMapsModule=$(find java-maps-* -name 'pom.xml'  \
 echo "Excluded modules: ${excludedMapsModule}"
 
 mvn clean deploy -B \
+  -T 1C \
+  -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss:SSS \
```

Comment:
@ddixit14 Can you add thread name? https://www.slf4j.org/api/org/slf4j/simple/SimpleLogger.html has `org.slf4j.simpleLogger.showThreadName`